app/mailers/visitor_mailer.rb

```diff
@@ -73,13 +67,11 @@ def first_date
     Date.parse(@visit.slots.first.date)
   end
 
-  def receipt_date
-    first_date.strftime('%e %B %Y').gsub(/^ /,'')
+  def slot_date
+    Date.parse(@slot.date)
   end
 
-  alias_method :rejection_date, :receipt_date
-
-  def confirmation_date
-    Date.parse(@slot.date).strftime('%e %B %Y').gsub(/^ /,'')
+  def format_date(date)
+    date.strftime('%e %B %Y').gsub(/^ /,'')
```

Comment:
This could also be a named format. Note that instead of `%e` with `gsub` to remove the padding, [we can use `%-d`](http://ruby-doc.org/core-2.0.0/Time.html#method-i-strftime):

```
%d - Day of the month, zero-padded (01..31)
        %-d  no-padded (1..31)
%e - Day of the month, blank-padded ( 1..31)
```